cmd/nvidia-container-runtime-hook/container_config.go

```diff
@@ -274,6 +276,14 @@ func getMigDevices(image image.CUDA, envvar string) *string {
 	return &devices
 }
 
+func getImexChannels(image image.CUDA) *string {
+	if !image.HasEnvvar(envNVImexChannels) {
+		return nil
+	}
+	chans := image.Getenv(envNVImexChannels)
+	return &chans
+}
+
```

Comment:
I know that we do this for `getMigDevices`, for example, but in this case I don't know whether we need to use `*string` as the return value. We could just return `image.Getenv(envNVImexChannels)` directly since if this is unset, this will be `""` and not be passed to the CLI anyway.

So:
```suggestion
func getImexChannels(image image.CUDA) string {
	return image.Getenv(envNVImexChannels)
}

```